YAXLib/Customization/IMemberContext.cs

```diff
@@ -11,19 +9,9 @@ namespace YAXLib.Customization;
 public interface IMemberContext
 {
     /// <summary>
-    /// The member's <see cref="MemberInfo" /> for member serialization, else <see langword="null" />.
-    /// </summary>
-    MemberInfo MemberInfo { get; }
-
-    /// <summary>
-    /// The member's <see cref="FieldInfo" /> for field serialization, else <see langword="null" />.
-    /// </summary>
-    FieldInfo? FieldInfo { get; }
-
-    /// <summary>
-    /// The member's <see cref="PropertyInfo" /> for property serialization, else <see langword="null" />.
+    /// The member's <see cref="IMemberDescriptor" /> for member serialization, else <see langword="null" />.
     /// </summary>
-    PropertyInfo? PropertyInfo { get; }
```

Comment:
Keep, but mark as "Obsolete, will be removed in a future version"

Reply:
Done. Please take a look.